scripts/purge-old-deployments

```diff
@@ -5,6 +5,8 @@ import logging
 import datetime
 import shutil
 
+from django.utils import timezone
```

Comment:
this script is intended to run without access to a virtualenv (/ django), so I think we should use the old codfe here.